drivers/i2c/i2c_mcux_lpi2c.c

```diff
@@ -202,6 +212,89 @@ static int mcux_lpi2c_transfer(const struct device *dev, struct i2c_msg *msgs,
 	return ret;
 }
 
+#if CONFIG_I2C_MCUX_LPI2C_BUS_RECOVERY
+static void mcux_lpi2c_bitbang_set_scl(void *io_context, int state)
+{
+	const struct mcux_lpi2c_config *config = io_context;
+
+	gpio_pin_set_dt(&config->scl, state);
+}
+
+static void mcux_lpi2c_bitbang_set_sda(void *io_context, int state)
+{
+	const struct mcux_lpi2c_config *config = io_context;
+
+	gpio_pin_set_dt(&config->sda, state);
+}
+
+static int mcux_lpi2c_bitbang_get_sda(void *io_context)
+{
+	const struct mcux_lpi2c_config *config = io_context;
+
+	return (gpio_pin_get_dt(&config->sda) != 0);
```

Comment:
nit: call return type is `int`, but this evaluates to a boolean

Reply:
Fixed, thanks.